### PR TITLE
Chore/release workflow

### PR DESCRIPTION
**Pull request by tjololo**
## Related Issue(s)
- #2 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.App.PlatformServices/Altinn.App.PlatformServices.csproj [hunk 1]
```diff
@@ -1,21 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```
Comment (RonnyB71) [inline src/Altinn.App.PlatformServices/Altinn.App.PlatformServices.csproj]: Do we need netcoreapp3.1?

Comment (tjololo) [reply src/Altinn.App.PlatformServices/Altinn.App.PlatformServices.csproj]: Did not support netcoreapp3.1 so it's removed

File: .github/workflows/publish-release.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        with:
+          dotnet-version: |
+            5.0.x
+            6.0.x
+      - name: Install deps
+        run: |
+          dotnet restore
+      - name: Build
+        run: |
+          dotnet build --configuration Release --no-restore -p:Deterministic=true -p:BuildNumber=${{ github.run_number }}
```
Comment (RonnyB71) [inline .github/workflows/publish-release.yml:27]: Do we need both build and pack? Won't pack also build? I see you have specified --no-build on pack, or do you want to know if the build fails specifically?

**Review by RonnyB71 — APPROVED** [review]
🕹️

File: .github/workflows/publish-release.yml (hunk 2 above)
Comment (tjololo) [reply .github/workflows/publish-release.yml:27]: Based the implementation on https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/.github/workflows/publish-packages-1.0.yml 
The idea is to more clearly display what part of the pack process (restore, build, pack) that fails.

**Pull request merged**